modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.tsx

```diff
@@ -173,11 +199,11 @@ const Main = ({
 	if (!multiple) {
 		if (
 			normalizedOptions.length &&
-			value[0] &&
+			value?.[0] &&
 			!normalizedOptions.find((option) => option.value === value[0])
 		) {
-			newValue = '';
-		}
+			newValue = undefined;
+		} //acho que eh pra limpar o campo quando eu deletar uma opcao do select
```

Comment:
same as comments above, check if you can change `undefined` to `null` and also remove the comments